cfgrammar/src/lib/span.rs

```diff
@@ -0,0 +1,41 @@
+/// A `Span` records what portion of the user's input something (e.g. a lexeme or production)
+/// references (i.e. the `Span` doesn't hold a reference / copy of the actual input).
+#[derive(Clone, Copy, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
```

Comment:
Just noticed an issue with this, I believe they need feature guards here for serde.

Reply:
Fixed, this only showed up when doing `cargo build` under `cfgrammar/`, because if I recall I believe one of the top-level crates uses the feature.